terror/terror.go

```diff
@@ -52,6 +57,12 @@ const (
 	CodeNoDataForHandle
 )
 
+// Variable error codes.
+const (
+	CodeUnknownStatusVar ErrCode = iota + 1
```

Comment:
ErrorCode should be unique in the same class, but doesn't have to be globally unique.